model/gen-ai/registry.yaml

```
@@ -474,3 +474,23 @@ groups:
                   "finish_reason": "stop"
                 }
               ]
+      - id: gen_ai.evaluation.name
+        stability: development
+        type: string
+        brief: The name of the evaluation used for the GenAI response.
+        examples: ["Relevance", "IntentResolution"]
+      - id: gen_ai.evaluation.score.value
+        stability: development
+        type: double
+        brief: The evaluation score returned by the evaluator.
+        examples: [4.0]
+      - id: gen_ai.evaluation.score.label
+        stability: development
+        type: string
+        brief: Human readable label for evaluation.
```

could you please add a note and explain what the label is? 

E.g.

```suggestion
        brief: Human readable label for evaluation.
        note: > 
          This attribute represents an interpretation of evaluation score when its applicable. For example, score value of 1 could mean that content is relevant when using one evaluator which measures score in range [-1, 1] and not relevant for another evaluator which measures it in the range [0, 5].
          This attribute SHOULD have low cardinality. Possible values depend on the evaluation metric and evaluator used. Implementations that report this attribute SHOULD document possible values.
```

I used my imagination and poor English to write this note, so please correct it and adjust to the real-life needs.

---

Thanks for the write up it helped a lot. Have added it to the note section